contents/blog/uh-i-like-my-job.md

```diff
@@ -0,0 +1,126 @@
+
+But getting there takes ongoing rigor, documented in shared artifacts that let everyone learn the business, not just their own job in the abstract.
+
+I know how PostHog wants to *win*, and how to do my part in it. Being rigorous prevents hand-waving from taking over the business.
+
+## 360 feedback
+
+I’m blessed and cursed with inspiring strong reactions from people. Some of them nice! Others more complicated.
+
+It has always been this way. In the typical organization, though, I’ve not found particularly strong release valves for strong opinions.
+
+A quarter into my second job, post-college, my boss blew his top at me, and we both spent a good half hour crying and talking it out.
+
+The alternative is deliberately-designed, ongoing 360 [feedback sessions](/newsletter/how-to-give-feedback) among teammates. 45 days into this job I got to hear candidly what people liked, what people wanted more of, and in a few cases, what people found grating in me.
+
+This gives me information to make better choices. I can evaluate the feedback and decide to lean further into places where my strengths are meshing with the team’s needs. I can make more intelligent tradeoffs between my ambitions and how my style impacts other people.
+
+Instead of being in the dark, I can navigate with confidence. Consistent feedback prevents resentments and poor direction from building up.
+
+## Fluid org chart
+
+The org chart is, in most places, treated like some holy truth carved in stone. It doesn’t readily change, except in calamitous, company-wide ruptures that change everyone’s lives overnight.
+
+Alternatively, you can build teams as [small squads](/handbook/company/small-teams), forming, dissolving, and combining talent as the needs of the business evolve.
+
+Every now and then I’ll look over my shoulder and see a new team has formed, cleaving off from previous work to start a new adventure.
+
+The flexibility has obvious advantages for the business, but for people, it’s an antidote to *stagnation.* The org chart evolves to suit the best fit of people on it, instead of trying to bend them into an imagined shape that doesn’t work. I can’t tell you how many people I’ve seen burn out because they couldn’t quite manage to become what someone randomly wrote down in a doc 18 months earlier.
+
+Instead, when we notice such a mismatch, a team’s *actual* strengths can be codified, and new teams can be chartered to fill any gaps. This is the luxury of a business that’s healthy and growing: slack in the system to also evolve.
+
+Even better, the org chart is descriptive, not prescriptive. It’s not a declaration of turf.
+
+I got to ship a new feature for our data pipelines product in my second week. *That’s not even my department*. So damn much fun.
```

Comment:
It might be fun to add a footnote on "That's not even my department" that explains what you're engineer working in marketing, and how crazy this. Can maybe talk about the job "Engineer who loves writing" and how that hooked you in. A meandering footnote can add color without messing up the narrative.

Reply:
I didn't know we had footnotes natively built into the site generator! Very into it.